src/main/java/com/ecom/inventoryService/inventoryService/controller/InventoryController.java

```diff
@@ -0,0 +1,56 @@
+package com.ecom.inventoryService.inventoryService.controller;
+
+
+import com.ecom.inventoryService.inventoryService.model.Inventory;
+import com.ecom.inventoryService.inventoryService.service.InventoryService;
+import jakarta.validation.Valid;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/inventory")
+@AllArgsConstructor
+@Slf4j
+public class InventoryController {
+
+    private InventoryService inventoryService;
+
+    @GetMapping
+    public ResponseEntity<List<Inventory>> getInventory() {
+        List<Inventory> inventory = inventoryService.getInventory();
+        if (!inventory.isEmpty()) {
+            return new ResponseEntity<>(inventory, HttpStatus.OK);
+        }
+        return new ResponseEntity<>(HttpStatus.NOT_FOUND);
```

Comment:
can we handle the empty list case from service layer and global exception handler?